airbyte-cdk/bulk/core/extract/src/main/kotlin/io/airbyte/cdk/read/StateManagerFactory.kt

```diff
@@ -202,6 +202,9 @@ class StateManagerFactory(
             if (cursorColumnID == CommonMetaField.CDC_LSN.id) {
```

Comment:
@postamar @xiaohansong , I don't fully understand CDC_LSN.
Is that still even a column?

Reply:
Mysql is only reading this field: https://github.com/airbytehq/airbyte/blob/c5c41e6389af470d0cc0e72da2568ff09ed3eac7/airbyte-integrations/connectors/source-mysql-v2/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlJdbcStreamFactory.kt#L43

Reply:
Thus I'm not sure if this fix is intended.

is CDC_CURSOR a shared cursor among all connectors? If so then yeah we should put it here and make sure oracle would follow this pattern. If not we probably need to overwrite this function in mysql.

Reply:
Ok. I think i get what you say. 
In general this is going to be CDC_LSN,
But mysql is using a different column. 
So we need to wire this to go to mysql's implementation